docs/en/ingest-management/release-notes/release-notes-8.10.asciidoc

```diff
@@ -0,0 +1,220 @@
+
+====
+
+[discrete]
+[[new-features-8.10.0]]
+=== New features
+
+The 8.10.0 release Added the following new and notable features.
+
+{fleet}::
+* Enable agent policy secret storage when all fleet servers are above 8.10.0 {kibana-pull}163627[#163627].
+* Kafka integration API {kibana-pull}159110[#159110].
+
+{fleet-server}::
+* Add a new policy token that can be used to enroll {agent} into fleet server. {fleet-server-pull}2654[#2654]
+* Add a Kafka output type for agent policies. {fleet-server-pull}2850[#2850]
+* Fleet Server support to handle agent policy secrets. {fleet-server-pull}2863[#2863] {fleet-server-issue}2485[#2485]
+
+{agent}::
+* Report the version from the {agent} package instead of the agent binary to enhance release process. {agent-pull}2908[#2908]
+* Implement tamper protection for {elastic-endpoint} uninstall use cases. {agent-pull}2781[#2781]
+* Add component-level diagnostics and CPU profiling. {agent-pull}3118[#3118]
+* Improve upgrade process to use upgraded version of Watcher to ensure a successful upgrade. {agent-pull}3140[#3140] {agent-issue}2873[#2873]
+
+[discrete]
+[[enhancements-8.10.0]]
+=== Enhancements
+
+{fleet}::
+* Add support for runtime fields. {kibana-pull}161129[#161129].
+
+{fleet-server}::
+* Keep the {fleet-server} service running when {es} is not available. {fleet-server-pull}2693[#2693] {fleet-server-issue}2683[#2683]
+* Add APM trace fields to HTTP request logs. {fleet-server-pull}2743[#2743]
+* File transfers with integrations now use datastreams. {fleet-server-pull}2743[#2741]
+* Use a unique ID for agent action results to ensure accurate counts on {fleet} UI. {fleet-server-pull}2782[#2782] {fleet-server-issue}2596[#2596]
+
+{agent}::
+* Redundant calls to `/api/fleet/setup` were removed in favor of {kib}-initiated calls. {agent-pull}2985[#2985] {agent-issue}2910[#2910]
+* Updated Go version to 1.20.7. {agent-pull}3177[#3177]
+* Add runtime prevention to prevent {elastic-defend} from running if {agent} is not installed in the default location. {agent-pull}3114[#3114]
+* Add a new flag `complete` to agent metadata to signal that the instance running is synthetics-capable. {agent-pull}3190[#3190] {fleet-server-issue}1754[#1754]
+* Add support for setting GOMAXPROCS to limit CPU usage through the agent policy. {agent-pull}3179[#3179]
+* Add logging to the restart step of the {agent} upgrade rollback process. {agent-pull}3245[#3245] {agent-issue}3305[#3305]
+
+[discrete]
+[[bug-fixes-8.10.0]]
+=== Bug fixes
+
+{fleet}::
+* Only show agent dashboard links if there is more than one non-server agent and if the dashboards exist. {kibana-pull}164469[#164469].
+* Exclude synthetics from per-policy-outputs. {kibana-pull}161949[#161949].
+* Fix the path for hint templates for auto-discover. {kibana-pull}161075[#161075].
+
+{agent}::
+* Don't trigger IOC alert on Windows uninstall. {agent-pull}3014[#3014] {agent-issue}2970[#2970]
```

Comment:
Suggestion from @cmacknz on this other PR: Indicator of Compromise (IOC)

Reply:
Thanks! I'll fix that via this PR: https://github.com/elastic/ingest-docs/pull/480